### ISSUE TITLE
x/pkgsite: Links on io/ioutil (and possibly other packages) are not readable

### ISSUE THREAD

**Issue by Manish-Giri** [golang/go#64387]
### What is the URL of the page with the issue?

https://pkg.go.dev/io/ioutil

### What is your user agent?

Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36

### Screenshot

![image](https://github.com/golang/go/assets/11348778/43ebe5a6-af66-44f4-9ce2-5a9f33e6e346)

### What did you do?

Visited documentation for io/ioutil pkg.

### What did you expect to see?

Readable links.

### What did you see instead?

Unreadable links.

**Comment by joeshin** [comment]
Did this get fixed somehow?  I'm seeing proper link colors in dark mode.

![Screen Shot 2024-03-21 at 9 50 53 PM](https://github.com/golang/go/assets/485843/c32330bf-894d-41d8-9a57-a9874f9c590e)

cc: @seankhliao

**Comment by Deleplace** [comment]
@joeshin I don't think this has been fixed, and I'm still seeing the white-on-yellow (unreadable) links.

There may be a different display color for "visited" or "unvisited" links, but I couldn't trigger a fifferent color by visiting the links.

**Comment by Deleplace** [comment]
I'm seeing the better colors in (Firefox, macOS) and in (Chrome, Android)

**Comment by joeshin** [comment]
@Deleplace @fzipp I was on the same combo of (MacOS Chrome).  And visited links looked fine as well.  Looking at CSS it appears that the "open" attribute is wonky somehow for some browsers?

![Screenshot 2024-05-07 at 11 57 34 AM](https://github.com/golang/go/assets/485843/28dc7d47-44a9-489d-aa4b-3784ec0e0678)
